graph-manager-docs/source/schema-validation.mdx

```diff
@@ -1,36 +1,200 @@
+3. Graph Manager returns the diff and a list of any breaking changes found.
+4. The Apollo CLI prints the result of the validation and returns a non-zero exit code if at least one breaking change is found.
+
+### The validation response
+
+Running `apollo service:check` outputs the diff of all detected schema changes  and highlights breaking changes:
+
+```console
+$ npx apollo service:check
+  ✔ Loading Apollo Project
+  ✔ Validated local schema against tag prod on service engine
+  ✔ Compared 8 schema changes against 110 operations over the last 24 hours
+  ✖ Found 3 breaking changes and 5 compatible changes
+    → breaking changes found
+
+FAIL    ARG_REMOVED                `ServiceMutation.checkSchema` arg `gitContext` was removed
+FAIL    FIELD_REMOVED              `Schema.fieldCount` was removed
+FAIL    FIELD_REMOVED              `Schema.typeCount` was removed
+
+PASS    FIELD_ADDED                `SchemaTag.schemaRepoID` was added
+PASS    FIELD_CHANGED_TYPE         `ServiceMutation.uploadPartialSchema` changed type from `UploadPartialSchemaResponse!` to `CompositionResult!`
+PASS    FIELD_DEPRECATION_REMOVED  `IntrospectionSchema.fieldCount` is no longer deprecated
+PASS    FIELD_DEPRECATION_REMOVED  `IntrospectionSchema.typeCount` is no longer deprecated
+PASS    TYPE_REMOVED               `UploadPartialSchemaResponse` removed
+
+View full details at: https://engine.apollographql.com/service/example-1234/check/<DETAILS>
+```
+
+Each change to the schema is labeled with either `PASS` or `FAIL`.
+
+> **Note:** Because breaking changes are detected by analyzing recent operations, your GraphQL server _must_ be [pushing analytics to Graph Manager](/setup-analytics/) for schema validation to work. **If there are no operation metrics to compare against, all schema changes are labeled with `PASS`.**
+
+The CLI output also includes a Graph Manager URL that provides full details on the changes and their impact on existing clients and operations:
+
+![Service check page in the Graph Manager UI](./img/schema-validation/service-check-page.png)
+
+> **Note:** If you've [installed schema validation checks on your GitHub PRs](#integrating-with-github), the "Details" link in your GitHub checks will take you to the same details link in this output.
+
+## Using with continuous integration
+
+Schema validation is especially useful when you add it to your continuous integration pipeline (e.g., Jenkins or CircleCI). By doing so, you can detect potential problems automatically and display validation results directly on pull requests.
+
+The `apollo service:check` command returns a non-zero exit code when it detects a breaking change, and it [integrates with GitHub statuses](#integrating-with-github), making it straightforward to use with most CI tools.
+
+Here's an example of how to add a schema validation check to CircleCI:
```

Comment:
We might want to specifically mention setting up the command as a separate job and capturing the markdown if they're not using GitHub in this section.

Reply:
We mention the tools of the exit code, but we don't say exactly how to use it, and the answer is "put it in another job" with the circle CI script as an example of that.

Reply:
I want to get this fix right and don't think I will on my own. I will bug you!